ui/pages/confirmations/components/confirm/info/batch/batch-simulation-details/batch-simulation-details.tsx

```diff
@@ -82,6 +77,14 @@ export function BatchSimulationDetails() {
     ];
   }, [approveBalanceChanges, handleEdit]);
 
+  if (
```

Comment:
Do we not need a similar check for the non-batch flow where we use `SimulationDetails` directly?

Reply:
Contract interactions are using batch-simulation-details components only.